src/Core/Configurations/RuntimeConfigValidator.cs

```diff
@@ -390,12 +396,13 @@ public static void ValidateGlobalEndpointRouteConfig(RuntimeConfig runtimeConfig
         /// <param name="runtimeConfig"></param>
         public static void ValidateRestURI(RuntimeConfig runtimeConfig)
         {
-            // CosmosDB_NoSQL does not support rest. No need to do any validations.
-            if (runtimeConfig.DataSource.DatabaseType is DatabaseType.CosmosDB_NoSQL)
+            if (runtimeConfig.DataSourceNameToDataSource.Values.Any(x => x.DatabaseType is DatabaseType.CosmosDB_NoSQL))
             {
+                // if we have a cosmos db even in multiple db scenario - no rest support.
```

Comment:
If the other dbs are SQL, why would we want to restrict REST support for those?

Reply:
In general are we allowing any REST experience for multiple datasources? i thought not and hence there should be no rest enabled for this scenario?